src/api/libcellml/namedentity.h

```diff
@@ -32,10 +32,10 @@ namespace libcellml {
 class LIBCELLML_EXPORT NamedEntity: public ParentedEntity
 {
 public:
-    ~NamedEntity() override; /**< Destructor. */
+    ~NamedEntity() override = default; /**< Destructor. */
     NamedEntity(const NamedEntity &rhs) = delete; /**< Copy constructor. */
     NamedEntity(NamedEntity &&rhs) noexcept = delete; /**< Move constructor. */
-    NamedEntity &operator=(NamedEntity rhs) = delete; /**< Assignment operator. */
+    NamedEntity &operator=(NamedEntity rhs) = delete; /**< Assignment operator, @private.*/
```

Comment:
Missing space at the end of the description.

Reply:
870b741